src/serializer/ResidualHeapSerializer.js

```diff
@@ -151,11 +151,35 @@ export class ResidualHeapSerializer {
     properties: Map<string, PropertyBinding> = obj.properties,
     objectPrototypeAlreadyEstablished: boolean = false
   ) {
-    /*
-    for (let symbol of obj.symbols.keys()) {
+    for (let [symbol, propertyBinding] of obj.symbols) {
       // TODO #22: serialize symbols
+      invariant(symbol instanceof SymbolValue);
+      invariant(propertyBinding.descriptor);
+      let propValue = propertyBinding.descriptor.value;
```

Comment:
Reading through the spec, I now realize that symbol properties can in fact have property getters and setters. So... The right course of action is in fact to adapt `_emitProperty`.